### ISSUE TITLE
Publish on Maven Central

### ISSUE THREAD

**Issue by penberg** [libsql/libsql-client-java#2]

**Comment by penberg** [comment]
I opened a ticket for a "org.libsql.client" group ID: https://issues.sonatype.org/browse/OSSRH-90878

**Comment by maxandersen** [comment]
any update now that OSSRH-90878 closed in April?

**Comment by penberg** [comment]
Hey @maxandersen! I haven't published anything on Maven central because nobody really asked for it. Also, I think we probably should just switch to a JDBC model for this driver instead to be more useful to people, which is why I haven't improve this.

**Comment by maxandersen** [comment]
Yeah. It was me looking for jdbc driver for Turso that got me here. 


Btw. Even using this manually didn't work for me - it gives me 401 for the https url with full token so was wondering if something changed or missing ?

**Comment by penberg** [comment]
Hey, works fine on my machine. Did you create the access token with `turso db tokens create <database name>`?

**Comment by maxandersen** [comment]
yes. no luck.
